CONTRIBUTING.md

```diff
@@ -25,9 +25,22 @@ You can help by improving the project's usage and developer instructions.
 
 ### Helping others
 
+You can help with code review, which reduces bugs, and over time has a
+wonderful side effect of making the code more readable and therefore more
+approachable. It's also a great way to teach and learn. Feel free to jump in!
+Be welcoming, appreciative, and helpful. Also please review
+[these impeccable guidelines][code review guidelines].
+
+You can perform first reviews of simple changes, like badge additions. You can
+also review @paulmelnikow's changes, most of which are otherwise self-
+reviewed.
```

Comment:
Github renders this text as "self- reviewed" (extra space after the dash) https://github.com/paulmelnikow/shields/blob/d3bf1322f3fc2d8230da1fd87853dd4599a4804b/CONTRIBUTING.md